packages/react-search-ui/src/SearchProvider.tsx

```diff
@@ -9,21 +9,15 @@ import {
 import SearchContext from "./SearchContext";
 
 import defaultA11yMessages from "./A11yNotifications";
+import { SearchDriverOptions } from "@elastic/search-ui/lib/esm/SearchDriver";
```

Comment:
Thanks! Rather than importing from a particular path, could you export the `SearchDriverOptions` type from the search-ui/types file?

```ts
export type { SearchDriverOptions } from "../SearchDriver";
```

then you can adjust the import path to be 

```ts
import { SearchDriverOptions } from "@elastic/search-ui";
```